tower-scripts/bin/rotate-log-access-key.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+set -o errexit
+
+# A script that generates SSH keys for opsmedic's ~/.ssh/authorized_keys
+# that will be used for cluster log gathering, and rotates them keeping
+# the last key there.
+#
+# The keys managed by this script look like this in authorized_keys:
+#
+#    command="gather-logs" ssh-rsa AAA...Ob logs_access_key_week_22-YMD-H:M:S
+#
+# where '22' is the week's number. The keys are expected to be rotated weekly,
+# so this is a way to check if there's already a key for this week. If there
+# is, the default behaviour is to not rotate the keys. This check can be
+# skipped  via the --force option or by setting FORCE=true in env.
+# The 'log_access_key_week' part of the comment can be changed via the
+# --prefix= option or the KEY_COMMENT_PREFIX env var.
+# The command associated with the key can be changed with the --command=
+# option or the KEY_COMMAND env var
+#
+# The new SSH private key is printed to stdout, meant to be collected by
+# a Jenkins pipeline running this script.
+
+# TODO: this script will be typically run from a hardcoded command= in
+# use-tower's authorized_keys. We should parse SSH_ORIGINAL_COMMAND
```

Comment:
Ok, I will just remove option parsing from the script and just leave the variables with pre-defined values.

Then I believe we don't need a pre-filter script, at least not to check arguments here - maybe for the logging side of things?